File: atcoder.jp/contests/abc287/tasks/abc287_a/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const N = +input();
const S = [...Array(N)].map(() => input());

const ans = N / 2 < S.filter((s) => s === "For").length;

console.log(ans ? "Yes" : "No");

File: atcoder.jp/contests/abc287/tasks/abc287_a/main.rs
use proconio::input;

fn main() {
    input! {
        n: usize,
        s: [String; n],
    }

    let mut cnt = 0;
    for s in s {
        if s == "For" {
            cnt += 1;
        }
    }

    let ans = n / 2 < cnt;

    println!("{}", if ans { "Yes" } else { "No" });
}
